rocky/rocky/views/mixins.py

```diff
@@ -46,6 +48,19 @@ class OriginData(BaseModel):
     boefje: Boefje | None = None
     params: dict[str, str] | None = None
 
+    @property
+    def is_old(self, maxage: Optional[int] = None) -> bool
+        if not maxage:
+            maxage = ORIGIN_MAX_AGE
+        if not self.normalizer:
+            return False
+            
+        observationdate = self.normalizer.get("raw_data", {}).get("boefje_meta", {}).get("ended_at", False)
+        old = datetime.datetime.utcnow() - datetime.timedelta(seconds=maxage)
+        if observationdate and observationdate < old:
+            return True
+        return False
```

Comment:
See above and this suggestion:
```suggestion
        return observation_date and observation_date < old:
```